packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -33,19 +35,41 @@ export default class SaytDriverPlugin implements Plugin {
   /**
    * Event name to listen for Sayt autocomplete requests.
    */
-  saytDataEvent: string = 'sfx::autocomplete_fetch_data';
+  autocompleteRequestEvent: string = 'sfx::autocomplete_fetch_data';
   /**
    * Event name for sending Sayt autocomplete responses.
    */
-  saytResponseEvent: string = 'sfx::autocomplete_received_results';
+  autocompleteResponseEvent: string = 'sfx::autocomplete_received_results';
   /**
    * Event name to listen for Sayt autocomplete errors.
    */
-  saytErrorEvent: string = 'sfx::autocomplete_sayt_error';
+  autocompleteErrorEvent: string = 'sfx::autocomplete_sayt_error';
+  /**
+   * Event name to listen for Sayt product requests.
+   */
+  productRequestEvent: string = 'sfx::sayt_products_request';
+  /**
+   * Event name for sending Sayt product responses.
+   */
+  productResponseEvent: string = 'sfx::sayt_products_response';
+  /**
+   * Event name to listen for Sayt product errors.
+   */
+  productErrorEvent: string = 'sfx::sayt_products_error';
+  /**
+   * Provide default configuration for SAYT product searches.
+   */
+  defaultSearchConfig: object = {
```

Comment:
Do we have an interface for this somewhere?

Reply:
@jmbtutor @DanielMoniz-GBI I see the same defaultSearchConfig object defined in both the sayt driver plugin and the search driver plugin.

Reply:
@jmbtutor what is your suggested direction on this?

Reply:
Is this not a `Partial<Request>`, `Request` coming from `groupby-api` (or in this case, `@sfx/search`? I haven't looked at the interface, though.

Reply:
@jmbtutor I can't tell if you're asking me to change anything here.

Reply:
@DanielMoniz-GBI Typing something as `object` is generally incorrect, so if we have an interface for it readily available (which I suspect is `Partial<Request>`; the search plugin should be re-exporing `Request`), I'd use that instead.